openpype/plugins/publish/collect_slate_global.py

```diff
@@ -0,0 +1,121 @@
+import os
+import json
+import pyblish.api
+from openpype import resources
+
+
+class CollectSlateGlobal(pyblish.api.InstancePlugin):
+    """Inject the data needed to generate Slates in the enabled families."""
+    label = "Collect for Slate Global workflow"
+    order = pyblish.api.CollectorOrder + 0.499
+    families = [
+        "review",
+        "render"
+    ]
+
+    def process(self, instance):
+
+        context = instance.context
+        slate_settings = context.data["project_settings"]["global"]\
+            ["publish"].get("ExtractSlateGlobal")
+
+        if not slate_settings:
+            self.log.warning("No slate settings found. Skipping.")
+            return
+
+        if not slate_settings["enabled"]:
+            self.log.warning("ExtractSlateGlobal is not active. Skipping.")
+            return
+
+        if context.data.get("host") == "nuke" and \
+                "render.farm" in instance.data.get("families"):
```

Comment:
Not sure what is reason of this condition?

But the key is `"hostName"` and is always filled.
```suggestion
        if (
            context.data["hostName"] == "nuke"
            and "render.farm" in instance.data["families"]):
```

Reply:
That's available only in pyblish pype UI, and does not reflect name of host integration but one of regitered host name in pyblish.

Reply:
fair enough, good to know!